mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -384,12 +388,12 @@ def _id_counter(self):
             yield i
             i += 1
 
-    def set_verbose_logging(self, verbose: bool = False):
+    def set_snippet_client_verbose_logging(self, verbose=False):
+        # type: (bool) -> None
         """Switches verbose logging. True for logging full RPC response.
 
         Args:
             verbose: bool, True for full logging full RPC response in DEBUG level.
```

Comment:
this sentence is a bit broken, rephrase.
Something like, "if True, turns on verbose logging; if False, turns off verbose loggin.